### PR TITLE
adds a skip locales flag to load v1 db command

### LINKED ISSUE #337: URL paths for localized pages when migrating from v1
In V1, localization is done differently:
- while we have it at the root level (locale prefix, e.g. /fr/...), in V1 the locale indication is at the leaf level (locale suffix, e.g. .../french-translation-of-[page name])

This adds an additional challenge to migrating localized content from V1 to V2.

- [x] As a first step, add a flag to `load_v1_db` to filter out pages from other locales (i.e. leave level slug is [language]-translation-of-[page name])
- [x] In the long run, automatically convert the whole page tree in a sensible manner

### PR DESCRIPTION
**Pull request by sheralim012**
Fixes #337 

- Added a skip-locales flag to load_v1_db management command
- Data of locales other than Default locale will be filtered out by giving above-mentioned flag

### CODE REVIEW

File: iogt_content_migration/management/commands/load_v1_db.py [hunk 1]
```diff
@@ -41,10 +41,16 @@ def add_arguments(self, parser):
             required=True,
             help='Path to IoGT v1 media directory'
         )
+        parser.add_argument(
+            '--skip-locales',
+            action='store_true',
+            help='Path to IoGT v1 media directory'
```
Comment (ehmadzubair) [inline iogt_content_migration/management/commands/load_v1_db.py:47]: change the help text

Comment (sheralim012) [reply iogt_content_migration/management/commands/load_v1_db.py:47]: done

**Comment by ehmadzubair** [comment]
👍🏼

**Comment by sheralim012** [comment]
@geoo89 kindly check and let me know

**Pull request merged**